Enterprise-CMCS/managed-care-review
# add helpdesk user with same perms as admin

## Summary

We want our helpdesk to have helpdesk user types in IDM, this adds them to the app with the same permissions as admin users.

#### Related issues

#### Screenshots

#### Test cases covered

Only one basic test was added so far for the parsing. 

## QA guidance

there should be a new helpdesk@example.com user you can log in with on the review app and confirm that you can see the same stuff an admin can see
